library/java/src/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -5,22 +5,29 @@
   public final String statsDomain;
   public final Integer connectTimeoutSeconds;
   public final Integer dnsRefreshSeconds;
```

Comment:
I considered putting all of the dns settings into a struct (same on iOS), but ultimately decided to keep this flat for this change.

I think if this lists keeps growing we need to first think about how we are going to manage config as it gets more unruly. But for now keeping it flat seemed the least intrusive change.

Reply:
sounds good